.circleci/config.yml

```diff
@@ -2823,15 +2823,6 @@ workflows:
           docker_image: "308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda10.2-cudnn7-py3-gcc7:9a3986fa-7ce7-4a36-a001-3c9bef9892e2"
           use_cuda_docker_runtime: "1"
           resource_class: gpu.medium
-      - pytorch_linux_test:
-          name: pytorch_linux_xenial_cuda10_2_cudnn7_py3_ge_config_profiling_test
```

Comment:
Should this be replaced with a `simple` version too?

Reply:
@Krovatkin can you take a look at this please? I hope I didn't mess this up during rebasing/merging?

Reply:
I added a cuda job to specifically test fuser. Fuser is disabled in the simple executor, so we should be good here.